coralproject/talk
# [CORL-233] Create the initial user history drawer

Building out the initial drawer that will need further work to add comment streams.

## What does this PR do?

Adds the user history drawer with the user's basic details (username, email, start date, id).

## How do I test this PR?

- Head to the moderation area
- Click on the username on a moderation card within one of the moderate queues
- User history drawer should appear
